src/index.ts

```diff
@@ -95,10 +95,30 @@ module.exports = class ServerlessPlugin {
 
   private async afterPackageFunction() {
     const config = getConfig(this.serverless.service);
+    const forwarderArn: string | undefined = config.forwarderArn;
+    const forwarder: string | undefined = config.forwarder;
+
+    let functionArn;
     if (config.enabled === false) return;
-    if (config.forwarder) {
+
+    if (forwarderArn && forwarder) {
+      console.error(
+        "Error: Could not add CloudWatch forwarder subscriptions because both 'forwarderArn' and 'forwarder' parameters are set. 'forwarderArn' and 'forwarder' are equivalent, please only use 'forwarderArn' or 'forwarder'.",
```

Comment:
Likewise here, I would encourage customers to specifically use `forwarderArn` if they have set both.